homeassistant/components/benq_projector/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Obtain and set state of a Benq projector over telnet-to-serial gateway.
+
+For more details about this component, please refer to the documentation
+at https://home-assistant.io/components/switch.benq_projector/
+"""
```

Comment:
The link to the documentation is no longer [needed](https://developers.home-assistant.io/docs/en/development_guidelines.html#file-headers).

Reply:
done